ArcGISRuntimeSDKQt_QMLSamples/Features/CreateMobileGeodatabase/CreateMobileGeodatabase.qml

```diff
@@ -0,0 +1,452 @@
+                anchors.rightMargin: 0
+                anchors.leftMargin: 0
+                enabled: gdbOpen
+                onClicked: {
+                    clearTable();
+                }
+            }
+
+            Button {
+                id: closeGdbButton
+                text: qsTr("Close .geodatabase")
+                anchors.left: parent.left
+                anchors.right: parent.right
+                anchors.rightMargin: 0
+                anchors.leftMargin: 0
+                enabled: gdbOpen
+                onClicked: {
+                    closeGeodatabase();
+                    gdbClosedNoticeRectangle.visible = true;
+                    buttonListRectangle.visible = false;
+                }
+            }
+
+            Text {
+                text: mobileGeodatabase ? "Created new geodatabase:\n" + mobileGeodatabase.path.toString().split("/").pop() : "Geodatabase path not found"
+                anchors.left: parent.left
+                anchors.right: parent.right
+                font.pixelSize: 12
+                wrapMode: Text.WrapAnywhere
+                anchors.rightMargin: 0
+                anchors.leftMargin: 0
+                visible: gdbOpen
+            }
+
+            Text {
+                text: "Number of features: " + numberOfFeatures + (numberOfFeatures === 0 ? "\n(Click or tap the map to add new features)" : "")
+                anchors.left: parent.left
+                anchors.right: parent.right
+                font.pixelSize: 12
+                wrapMode: Text.WordWrap
+                anchors.rightMargin: 0
+                anchors.leftMargin: 0
+                visible: gdbOpen
+            }
+        }
+    }
+
+    Rectangle {
+        id: gdbClosedNoticeRectangle
+        anchors.centerIn: parent
+        width: parent.width * 0.75
+        height: gdbInfoColumn.height + 20
+        color: "white"
+        border.color: "black"
+        clip: true
+        visible: false
+
+        // Prevent mouse interaction from propagating to the MapView
+        MouseArea {
+            anchors.fill: parent
+            onPressed: mouse.accepted = true;
+            onWheel: wheel.accepted = true;
+        }
+
+        Column {
+            id: gdbInfoColumn
+            anchors.centerIn: parent
+            anchors.margins: 10
+            spacing: 10
+            width: parent.width - 20
+            height: children.height
+
+            Text {
+                id: gdbNameText
+                width: parent.width
+                text: "Closed and saved geodatabase to the temporary path:"
+                wrapMode: Text.WordWrap
+            }
+            TextEdit {
+                id: gdbPathText
+                width: parent.width
+                readOnly: true
+                selectByMouse: true
+                text: mobileGeodatabase ? mobileGeodatabase.path : "The geodatabase returned no path property"
+                wrapMode: Text.WrapAnywhere
+            }
```

Comment:
Because C++ and QML manipulate paths separately I'm inclined to leave them as is. Trying to format each for consistency may get complicated, produce unintended consequences, and ultimately not provide much value for the user in my opinion.